warehouse/utils/db/types.py

```diff
@@ -11,7 +11,12 @@
 # limitations under the License.
 import datetime
 
-from sqlalchemy import DateTime, TypeDecorator
+from typing import Annotated
+
+from sqlalchemy import Boolean, DateTime, TypeDecorator, sql
```

Comment:
```suggestion
from sqlalchemy import DateTime, TypeDecorator, sql
```

Reply:
I don't understand why this would be removed.

Reply:
gotcha, it was dependent on another suggestion. cb233f632